easybuild/easyconfigs/l/LWP-Protocol-https/LWP-Protocol-https-6.07-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,72 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'Bundle'
+
+name = 'LWP-Protocol-https'
+version = '6.07'
+
+homepage = 'https://github.com/libwww-perl/lwp-protocol-https'
+description = "Provide https support for LWP::UserAgent"
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+
+builddependencies = [('binutils', '2.32')]
+dependencies = [
+    ('Perl', '5.30.0'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('LWP::UserAgent', '6.06', {
+        'source_tmpl': 'libwww-perl-6.43.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/O/OA/OALDERS'],
+        'checksums': ['e9849d7ee6fd0e89cc999e63d7612c951afd6aeea6bc721b767870d9df4ac40d'],
+    }),
+    ('Mozilla::CA', '20110101', {
+        'source_tmpl': 'Mozilla-CA-20180117.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/A/AB/ABH'],
+        'checksums': ['f2cc9fbe119f756313f321e0d9f1fac0859f8f154ac9d75b1a264c1afdf4e406'],
+    }),
+    ('Net::HTTPS', '6', {
+        'source_tmpl': 'Net-HTTP-6.19.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/O/OA/OALDERS'],
+        'checksums': ['52b76ec13959522cae64d965f15da3d99dcb445eddd85d2ce4e4f4df385b2fc4'],
+    }),
+    ('Net::HTTPS', '6', {
+        'source_tmpl': 'Net-HTTP-6.19.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/O/OA/OALDERS'],
+        'checksums': ['52b76ec13959522cae64d965f15da3d99dcb445eddd85d2ce4e4f4df385b2fc4'],
+    }),
+    ('Test::More', '0', {
+        'source_tmpl': 'Test-Simple-1.302171.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/E/EX/EXODIST'],
+        'checksums': ['e27f90d2b2a6bc6ffa7675a072c2f41d5caffd99858dc69b2030940cc138368a'],
+    }),
+    ('Test::RequiresInternet', '0', {
+        'source_tmpl': 'Test-RequiresInternet-0.05.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MALLEN/'],
+        'checksums': ['bba7b32a1cc0d58ce2ec20b200a7347c69631641e8cae8ff4567ad24ef1e833e'],
+    }),
+    ('LWP::Protocol::https', version, {
```

Comment:
maybe this whole thing can be added to `Perl`, since it's all in CPAN anyway?

Reply:
moving to Perl